client-v2/src/shared/components/input/InputImage.tsx

```diff
@@ -168,9 +170,11 @@ class InputImage extends React.Component<ComponentProps, ComponentState> {
       gridUrl,
       useDefault,
       defaultImageUrl,
-      message = 'Add image'
+      message = 'Add image',
+      editMode
     } = this.props;
-    const gridSearchUrl = `${gridUrl}?cropType=landscape`;
+    const gridSearchUrl =
+      editMode === 'editions' ? `${gridUrl}` : `${gridUrl}?cropType=landscape`;
```

Comment:
`gridUrl` is of type `string | null` and `gridSearchUrl` needs to be a string... so ``${gridUrl}` rather than just `gridUrl`.